src/scss/typography.scss

```diff
@@ -14,46 +48,149 @@ p,
 .lead,
 ul,
 ol {
-  margin-bottom: $line-height-computed * .75;
+  margin-bottom: $line-height-computed * .5;
 }
 
+h1,
+.h1,
+h2,
+.h2,
+h3,
+.h3,
+h4,
+.h4,
 h5,
 .h5,
 h6,
 .h6 {
-  line-height: $line-height-base;
+  margin-top: $line-height-computed;
+}
+
+h1,
+.h1 {
+  font-size: 42px;
```

Comment:
All of these font sizes need to be variables

Reply:
Updated to use variables 👍